tools/deeptools.wdl

```diff
@@ -21,13 +30,13 @@ task bam_coverage {
         String prefix = basename(bam, ".bam")
         Boolean use_all_cores = false
         Int ncpu = 1
-        Int memory_gb = 5
+        Int memory_gb = 4
         Int modify_disk_size_gb = 0
         Int max_retries = 1
     }
 
     Float bam_size = size(bam, "GiB")
-    Int disk_size_gb = ceil(bam_size * 4) + 10 + modify_disk_size_gb  # is all this disk needed?
+    Int disk_size_gb = ceil(bam_size * 4) + 10 + modify_disk_size_gb  # TODO is all this disk needed?
```

Comment:
I think scaling back to `* 1.5` is fine for now. (as in right now, on this PR).
I opened an issue (#88) to investigate all our disk sizes (because I don't have confidence they are all set sanely). If 1.5 scale is wrong we can address it then instead of holding up this PR over it.